tutorial_scripts/01_quickstart.py

```diff
@@ -0,0 +1,29 @@
+import weave
```

Comment:
This entire directory is a copy of the code in the docs. Ideally these are programmatically kept in sync, but this was really good for testing manually running through. Also, this is nice for demoing.

Reply:
At some point (hackweek?) I can update the wf tool to work with this directory.